tests/test_gh.py

```
@@ -0,0 +1,20 @@
+from scikit_package.cli.gh import _broadcast_issue_to_urls  # noqa: F401
+
+
+def test_broadcast_issue_to_urls():
+    # C1: complete issue_content, a list of target repo urls and
+    #   dry_run is True.
+    #   Expect status_flag to be 1. Issues are not created in the
+    #   target repos.
+    # C2: complete issue_content, a list of target repo urls, and
+    #   dry_run -s False.
+    #   Expect status_flag to be 0. Issues are created in the target
+    #   repos.
+    assert False
+
+
+def test_broadcast_issue_to_urls_bad():
+    # C1: no "title" or "body" key found in the issue_content.
```

I am thinking whether we should move to a more advanced "error" since we are developing an app? Like using the `logging` module. The interface is cleaner, and less nasty debug information is exposed to the user.

We can start it from this functionality and gradually use `logging` to replace other `ValueError`, `FileNotFoundError`, ...

---

we could go with logging, but we can also use `warn` which is probably more appropriate for task, is it?

---

Yes. For missing "title" or "body", we can raise an Error since they are required to create issues. In other cases I will use `warnings`.